activiti-cloud-service-common/activiti-cloud-services-test/src/main/java/org/activiti/cloud/starters/test/builder/ProcessInstanceEventContainedBuilder.java

```diff
@@ -70,4 +70,17 @@ public ProcessInstanceImpl aRunningProcessInstanceWithStartDate(String name, Dat
         return processInstance;
     }
 
+    public ProcessInstanceImpl aRunningProcessInstanceWithCompletedDate(String name, Date completedDate) {
+        ProcessInstanceImpl completedProcess = new ProcessInstanceImpl();
```

Comment:
You can use `buildProcessInstance(name)` to avoid all the field settings except `CompleteDate`